wdtk-client/src/main/java/org/wikidata/wdtk/client/RdfSerializationAction.java

```diff
@@ -251,4 +254,21 @@ private void printHelp() {
 					+ TASK_HELP.get(supportedTask), 75, "\n   ", true));
 		}
 	}
+
+	@Override
+	public String getReport() {
+		String message = "Finished serialization of "
+				+ this.serializer.getTripleCount() + " RDF triples in file "
+				+ this.outputDestination;
+		return message;
+	}
+
+	@Override
+	public String getActionName() {
```

Comment:
We could also use a Getter for an default action name. So we can move the method to the abstract super class and the default action name need not depend on the class name.